Interaction - Mission: Cauterize, Uncertain Destinations

claywar opened the pull request:
<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
<!-- (it should look like this: - [x] I have ...) -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Implements RoV Mission: Cauterize
<!-- Describe what your PR does here. If it closes an existing issue, you can mention: "Closes #1234" and GitHub will link this PR to that issue. -->

## Steps to test these changes
Follow guides for this mission, along with notes contained within the mission script.
<!-- Clear and detailed steps to test your changes here -->

neuromancerxi commented on sql/npc_list.sql:
```diff
@@ -12849,7 +12849,7 @@ INSERT INTO `npc_list` VALUES (17207824,'qm4','???',102,426.315,8.563,-163.337,1
 INSERT INTO `npc_list` VALUES (17207826,'Wolfgang','Wolfgang',0,0.000,0.000,0.000,0,40,40,0,1,0,6,27,0x0100020114101920193019401950006000700000,32,NULL,1);
 -- airship
--- INSERT INTO `npc_list` VALUES (17207827,'','',0,0.000,0.000,0.000,0,40,40,19,0,0,2,3,0x040000000B000000000000000000000000000000,32,null,1);
+INSERT INTO `npc_list` VALUES (17207827,x'0B','NPC',0,0.000,200.000,0.000,65537,50,50,19,0,0,0,3,0x0400000000000000000000000000000000000000,0,NULL,1);
```
Stray x outside of single quotes.

WinterSolstice8 replied:
That's intentional, x'0B' = character value of 0x0B, it's used for non-printable NPC names, see https://github.com/LandSandBoat/server/pull/2483 for a more obvious example of what this means

neuromancerxi replied:
Thanks for the clarification, first time I've come across it so it stood out as a potential typo to me.

WinterSolstice8 replied:
Oh yeah, it certainly is weird isn't it? It took me a good while to figure out how to use non-printable characters in the db. Very counter-intuitive compared to most other languages that lets you escape in string such as "\x0B"

claywar committed:
Interaction - Mission: Cauterize, Uncertain Destinations

Initial blocking

Fill in messaging, standardize NPC names


Complete mission, add additional notes from Goblin Footprint


Formatting


Move ID local to relevant block


CI issues


ID


Fomatting to match tooling


Add Lightsworm KI to appropriate missions

WinterSolstice8 commented on scripts/zones/Batallia_Downs/IDs.lua:
```diff
@@ -21,6 +21,7 @@ zones[xi.zone.BATALLIA_DOWNS] =
         CARRIED_OVER_POINTS      = 7023,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7024,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
"ou" are unable

claywar replied:
See: https://github.com/zach2good/UpdateExtractor/tree/main/out
This is intentional.

WinterSolstice8 commented on scripts/zones/Batallia_Downs_[S]/IDs.lua:
```diff
@@ -18,6 +18,8 @@ zones[xi.zone.BATALLIA_DOWNS_S] =
         CARRIED_OVER_POINTS     = 7001, -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER            = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS      = 7047, -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
"ou" are unable

WinterSolstice8 commented on scripts/zones/Rolanberry_Fields/IDs.lua:
```diff
@@ -20,6 +20,7 @@ zones[xi.zone.ROLANBERRY_FIELDS] =
         CARRIED_OVER_POINTS      = 7023,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7024,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
"ou" are unable

WinterSolstice8 commented on scripts/zones/Rolanberry_Fields_[S]/IDs.lua:
```diff
@@ -17,6 +17,8 @@ zones[xi.zone.ROLANBERRY_FIELDS_S] =
         CARRIED_OVER_POINTS     = 7001, -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER            = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS      = 7047, -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
"ou" are unable

WinterSolstice8 commented on scripts/zones/Sauromugue_Champaign/IDs.lua:
```diff
@@ -20,6 +20,7 @@ zones[xi.zone.SAUROMUGUE_CHAMPAIGN] =
         CARRIED_OVER_POINTS      = 7023,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7024,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
"ou" are unable

WinterSolstice8 commented on scripts/zones/Sauromugue_Champaign_[S]/IDs.lua:
```diff
@@ -17,6 +17,8 @@ zones[xi.zone.SAUROMUGUE_CHAMPAIGN_S] =
         CARRIED_OVER_POINTS      = 7001,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7002,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7003,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7047,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
"ou" are unable

WinterSolstice8 reviewed (changes requested):
Just a few missing "Y"s... 👍

WinterSolstice8 reviewed (approved):
missing "Y" is from some bit magic corrupting the first character, so LGTM in that case
